src/compiler/parser.ts

```diff
@@ -7658,6 +7690,13 @@ namespace ts {
                     return finishNode(factory.createJSDocTypeTag(tagName, typeExpression, comments), start, end);
                 }
 
+                function parseSeeTag(start: number, tagName: Identifier, indent?: number, indentText?: string): JSDocSeeTag {
+                    const nameExpression = tryParseJSDocNameExpression();

```

Comment:
Seems that should not be an error (the same as `@type` tag)

Reply:
As the result. `unittests - parsesCorrectly - @link tags` failed.
Because `@see {@link second link text} and {@link Foo|a foo} as well.` cannot work weel.

Reply:
oh, that makes sense; `@see {@link ... }` is a pretty common construction I think. never mind.